src/compiler/binder.ts

```diff
@@ -2154,12 +2154,11 @@ namespace ts {
                 transformFlags = TransformFlags.AssertES6;
 
                 // A MethodDeclaration is TypeScript syntax if it is either async, abstract, overloaded,
-                // generic, or has both a computed property name and a decorator.
+                // generic, or has a decorator.
                 if ((<MethodDeclaration>node).body === undefined
                     || (<MethodDeclaration>node).typeParameters !== undefined
                     || hasModifier(node, ModifierFlags.Async | ModifierFlags.Abstract)
-                    || (subtreeFlags & TransformFlags.ContainsDecorators
-                        && subtreeFlags & TransformFlags.ContainsComputedPropertyName)) {
+                    || subtreeFlags & TransformFlags.ContainsDecorators) {

```

Comment:
No, previously this only set `AssertTypeScript` if the method contained *both* decorators and a computed property name. Now it sets it if it contains *at least* decorators.